### PR TITLE
Prep 0.7.0 release

### PR DESCRIPTION
**Pull request by imback82**
This PR prepares the v0.7.0 release.

### CODE REVIEW

File: docs/release-notes/0.7/release-0.7.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+* Expose `Column.Apply()` ([#323](https://github.com/dotnet/spark/pull/323))
+* "Deprecated" annotation is added to the following functions:
+  * `WindowSpec.RangeBetween(Column start, Column end)`
+  * `Window.RangeBetween(Column start, Column end)`
+  * `Functions.UnboundedPreceding()`
+  * `Functions.UnboundedFollowing()`
+  * `Functions.CurrentRow()`
+
+### Breaking Changes
+* None
```
Comment (suhsteve) [inline docs/release-notes/0.7/release-0.7.md]: Should the removal of *ActiveSession() APIs be mentioned also ?

Comment (imback82) [reply docs/release-notes/0.7/release-0.7.md]: Yes! let me update it.

File: docs/building/windows-instructions.md [hunk 2]
```diff
@@ -242,7 +242,7 @@ Once you build the samples, running them will be through `spark-submit` regardle
          --jars path\to\net.jpountz.lz4\lz4-1.3.0.jar,path\to\org.apache.kafka\kafka-clients-0.10.0.1.jar,path\to\org.apache.spark\spark-sql-kafka-0-10_2.11-2.3.2.jar,`path\to\org.slf4j\slf4j-api-1.7.6.jar,path\to\org.spark-project.spark\unused-1.0.0.jar,path\to\org.xerial.snappy\snappy-java-1.1.2.6.jar `
          --class org.apache.spark.deploy.dotnet.DotnetRunner `
          --master local `
-         C:\github\dotnet-spark\src\scala\microsoft-spark-2.3.x\target\microsoft-spark-2.3.x-0.6.0.jar `
+         C:\github\dotnet-spark\src\scala\microsoft-spark-2.3.x\target\microsoft-spark-2.3.x-0.7.0.jar `
```
Comment (suhsteve) [inline docs/building/windows-instructions.md]: Nit: For the ubuntu and window instructions, do you want to just replace this with `<version>` so you don't have to update these files again with subsequent releases ?

Comment (imback82) [reply docs/building/windows-instructions.md]: done

**Pull request merged**